src/Sylius/Bundle/ResourceBundle/EventListener/AbstractDoctrineSubscriber.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ResourceBundle\EventListener;
+
+use Doctrine\Common\EventSubscriber;
+use Doctrine\Common\Persistence\Mapping\ClassMetadata;
+use Sylius\Component\Resource\Metadata\RegistryInterface;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
```

Comment:
I am not sure if it is true for Symfony (probably no), but in Sylius we should definitely only mention the real authors of the file, not me everywhere. :)